package.json

```diff
@@ -14,6 +14,7 @@
     "lint:fix": "standard --fix",
     "lint:standard": "standard --verbose | snazzy",
     "lint:typescript": "eslint -c types/.eslintrc.json types/**/*.d.ts test/types/**/*.test-d.ts",
+    "prepublishOnly": "tap --no-check-coverage test/internals/version.test.js",
```

Comment:
I think the `build/sync-version.js` should run in here?

Reply:
If a maintainer has already made a git commit, that could make the git tree dirty, and potentially create mismatching sources (between git and npm).

Reply:
Actually, fastify already requires commits to pass tests (incl. this version check one) before they are made into the git tree. The pre-publish check is just an additional safeguard.

In any case, the sync-version script can't be run automatically before or after any action, as that could lead to unexpected uncommitted changes. It has to be run by a maintainer.

Reply:
For the main repo, I think no body other than mcollina will release a new version. If he is ok with it, that is fine.

Reply:
Oh. Now I remember, there is one for the job: `version` hook for npm.

> 3. Bump version in package.json as requested (patch, minor, major, etc).
> 4. Run the version script. These scripts have access to the new version in package.json (so they can incorporate it into  file headers in generated files for example). Again, scripts should explicitly add generated files to the commit using git add.
> 5. Commit and tag.
> 6. Run the postversion script. Use it to clean up the file system or automatically push the commit and/or tag.

https://docs.npmjs.com/cli/v7/commands/npm-version

Reply:
Judging from the commit title (`x.x.x` vs `Bumped vx.x.x` of fastify), though, I don't think @mcollina is using `npm version`, so this probably won't be useful.